reaskapi/deepcyc.py

```diff
@@ -4,14 +4,16 @@
 
 class DeepCyc(ApiClient):
 
-    def __init__(self, config_section='default'):
+    def __init__(self, config_section='default', product_version='DeepCyc-2.0.7'):
```

Comment:
any reason to set a specific version as default instead of `None`?
If it sets a specific default version, we need to update `api` repo every time we update default version in api-server side.